src/transports.jl

```diff
@@ -93,13 +94,20 @@ function read!(t::TFramedTransport, buff::Vector{UInt8})
     end
     buff
 end
-function read(t::TFramedTransport, UInt8)
+function read(t::TFramedTransport, ::Type{UInt8})
```

Comment:
From testing, I realized that the function is sometimes called with other unsigned types and so I did this with SocketBase:

https://github.com/tanmaykm/Thrift.jl/blob/fb8baf9849dce745dc8c158bd45c9aa20ff9a226/src/transports.jl#L208

Reply:
Yes, makes sense to generalize. Updated in https://github.com/tanmaykm/Thrift.jl/pull/72/commits/fb5b83b9e153c1f30df10d856716eb90da1554a5